src/datomic_toolbox/core.clj

```diff
@@ -21,97 +20,28 @@
 
 (defn db [] (d/db (connection)))
 
+(defn tempid
+  ([]  (d/tempid (partition)))
+  ([n] (d/tempid (partition) n)))
+
 (defn transact [tx-data]
```

Comment:
Do you mean `transact` or `tempid` or both? I went back and forth on this too. The reason they are still in here is that they are examples of the stateful `datomic.api` replacement functions that make up `core` now. In the end, I liked having all of those in core vs. having just one or two in a different place. That way if you're calling a non-core fn, you know it is isn't using state from `configure!` or `initialize`.